openpype/pipeline/legacy_io.py

```diff
@@ -30,7 +30,7 @@ def install():
 
     session = session_data_from_environment(context_keys=True)
 
-    session["schema"] = "openpype:session-3.0"
+    session["schema"] = "openpype:session-4.0"
```

Comment:
Should really be `ayon:session-1.0` considering that is the reason for it's update.

Reply:
I don't know if we should keep this schema for ayon to be honest.